nginx_configs/albs.conf

```diff
@@ -58,6 +58,10 @@ server {
     }
 
     location /pulp/ {
+        # header for decompressing build/test logs
+        if ($request_uri ~* (\.log$)) {
```

Comment:
not .gz?

Reply:
we discussed it with Vasiliy and we decided to left it like it is right now, because .gz at the end means that it would be an archive, I tried to keep archives in our storages, but with this nginx header it will decompress automatically and user will lost ability to open it via file manager